include/psp2kern/vfs.h

```diff
@@ -0,0 +1,810 @@
+/**
+ * \kernelgroup{SceVfs}
+ * \usage{psp2kern/vfs.h,SceIofilemgrForDriver_stub}
+ */
+
+#ifndef _PSP2KERN_VFS_H_
+#define _PSP2KERN_VFS_H_
+
+#include <psp2common/kernel/iofilemgr.h>
+#include <psp2kern/kernel/threadmgr/fast_mutex.h>
+#include <psp2kern/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Forward declarations
+typedef struct SceVfsMount SceVfsMount;
+typedef struct SceVfsVnode SceVfsVnode;
+typedef struct SceVfsFile SceVfsFile;
+typedef struct SceVfsOpTable SceVfsOpTable;
+typedef struct SceVopTable SceVopTable;
+
+/**
+ * @name VFS Data Types
+ */
+/** @{ */
+#define SCE_VFS_TYPE_FS    (0x00000000)
+#define SCE_VFS_TYPE_DEVFS (0x00000010)
+
+/**
+ * VFS Info defining a VFS Implementation
+ */
+typedef struct SceVfsInfo {
+    SceVfsOpTable *vfsOps;    //!< VFS Implementation operations for mountpoint actions.
```

Comment:
can you change these member names to `snake_case`?

Reply:
same as all structures

Reply:
Sure I can do it, as I understand it's for consistency reasons in the SDK, right? The current camelCase is derived from debug strings referencing the fields.

Reply:
yeah. i know. couple years ago, princess was also explained that.

it just sdk naming reason :)